iron-list.html

```diff
@@ -1254,6 +1254,7 @@
         var inst = el._templateInstance;
         var item = this.items && this.items[vidx];
         if (item != null) {
+          inst[this.as] = {};
```

Comment:
I see. However, this will make bindings run with `undefined` values. see this thread: https://github.com/PolymerElements/iron-list/pull/328#discussion_r82298950